### PR TITLE
Shows detail info about the covered tests

### PR DESCRIPTION
**Pull request by LinkaiQi**
Refer to Issue #45 
Now, the list of the tests in the right-sidebar is expandable, and has a new look.

screen shot 2017-11-20 at 9 48 17 pm

Try: http://localhost:3000/file?revision=c616a6fd5e4b&path=/accessible/atk/Platform.cpp

### CODE REVIEW

**Comment by klahnakoski** [comment]
that looks nicer!

**Comment by klahnakoski** [comment]
when the test name is collapsed, can the platform be hidden?  That way the suite/chunk is easier to read.

File: src/components/fileviewercov.js [hunk 1]
```diff
@@ -45,16 +46,56 @@ export const TestsSideViewer = ({ coverage, lineNumber }) => {
...
+class Test extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { height: '0px', ptrRotate: 0 };
+    this.handleTestOnClick = this.handleTestOnClick.bind(this);
+  }
+
+  handleTestOnClick() {
+    this.setState({
+      height: (this.state.height === '0px') ? '60px' : '0px',
```
Comment (klahnakoski) [inline src/components/fileviewercov.js]: Instead of changing height, change the style class.

**Comment by LinkaiQi** [comment]
> when the test name is collapsed, can the platform be hidden? That way the suite/chunk is easier to read.

Are you talking about when the test name is too long to display and the remaining letters are replaced by "..." ? (e.x. ```test-linux64-ccov/opt-talos-tp6-stylo-...```). 
In this case, hide the platform part?

**Comment by yuenj** [comment]
looks good to me

**Pull request merged**